### DISCUSSION TITLE
Classic Explorer Bar - Lag

### DISCUSSION THREAD

**Discussion by Torgut** [Open-Shell/Open-Shell-Menu#1871] [General]
Anytime I open File Explorer in Windows 11 the Classic Explorer Bar will cause lag on rendering the window. I can then turn off the bar and minimize and maximize that window will be fine. But as soon as I open a new File Explorer window the problem will happen again. Any way to disable this Classic Explorer Bar for good?

**Comment by ge0rdi** [comment]
> Any way to disable this Classic Explorer Bar for good?

You can uninstall the feature by running `Open-Shell` installer again.
